### ISSUE TITLE
How do you connect to the database via a GUI?

### ISSUE THREAD

**Issue by madisonmanhire** [markshust/docker-magento#909]
For example, Warden has a page in their documentation that shows settings for connecting to popular database GUIs like TablePlus: https://docs.warden.dev/configuration/database.html

Are there similar settings required for connecting one of these tools to Docker Magento's database?

**Comment by artmouse** [comment]
Use 127.0.0.1 as host for mysql.

**Comment by bbakalov** [comment]
@madisonmanhire check how it is configured in my local. 
You don't need any tunnel for this connection. 

<img width="835" alt="image" src="https://user-images.githubusercontent.com/9625352/235872236-7b53907a-6c30-4eec-b859-af4945bb1341.png">

**Comment by madisonmanhire** [comment]
I figured out my issue here. The above recommendations weren't working for me because I had MySQL installed elsewhere on my computer. I needed to update the port in `compose.yaml:47` to something other than 3306 (in this case, 3356), then use that new port number in TablePlus.

```
ports:
  - "3356:3306"
```

<img width="508" alt="Screenshot 2023-06-19 at 1 50 53 PM" src="https://github.com/markshust/docker-magento/assets/9360473/e9bf4c93-643c-4e28-858a-8fb3199bc177">
